lib/signaling/v2/peerconnection.js

```diff
@@ -472,9 +467,7 @@ class PeerConnectionV2 extends StateMachine {
         // support, we have to use the same hacky solution as Safari. Revisit
         // this when RTCRtpTransceivers and MIDs land. We should be able to use
         // the same technique as Firefox.
-        : isSafari || this._isUnifiedPlan
-          ? new OrderedTrackMatcher()
-          : new IdentityTrackMatcher();
+        : isSafari || isUnifiedPlan ? new OrderedTrackMatcher() : new IdentityTrackMatcher();
```

Comment:
@syerrapragada 

Yes, because the ternary expression is short enough to be in a single line. Also, `isUnifiedPlan` is now calculated once on load and not for every instance of `PeerConnectionV2`.